### ISSUE TITLE
eos_designs: 7280R/R2 platform default mlag reload delay too low

### ISSUE THREAD

**Issue by kmueller68** [aristanetworks/ansible-avd#1046]
## Issue Type

- [x] Bug Report
- [ ] EOS Configuration rendering error
- [ ] Documentation Issue Report

## Summary

I have experienced a topic during EOS updates done by CVP change control yesterday.
I included the CVP MLAG Health Check in CC but it failed with all 7280R/R2 switches.

I recognized the "show mlag issu warnings" in the logs that the reload-delay is too low.
```bash
show mlag issu warnings
The configured reload delay of 780 seconds is below the recommended
value of 900 seconds. A longer reload delay allows more time to
rollback an unsuccessful upgrade due to incompatibility.
```

The current AVD default value for 7280R/R2 is at 780 seconds but should be at least at 900 seconds based on the latest EOS.

```yaml
platform_settings:
    <snip>
  - platforms: [ 7800R3, 7500R3, 7500R, 7280R3, 7280R2, 7280R ]
    tcam_profile: vxlan-routing
    lag_hardware_only: true
    reload_delay:
      mlag: 780
      non_mlag: 1020
```

Could you please doublecheck this settings, if it would be useful to adjust your default values too.

## Component Name
eos_designs

## Steps to reproduce

```yaml

```

### Expected results

```shell
$
```

### Actual results

```shell

```

**Comment by carlbuchmann** [comment]
@kmueller68 - Thanks for raising this issue, I will investigate and report back and adjust this value accordingly!

**Comment by c-po** [comment]
Hi @carlbuchmann did you had any chance checking this one out in the meantime?

Just got hit by this one while running a MLAG health check.

```
mlag configuration
   domain-id leaf-2g6.SSN.MU
   local-interface Vlan4094
   peer-address 172.29.24.13
   peer-link Port-Channel501
   reload-delay mlag 780
   reload-delay non-mlag 1020
!
```

```
leaf-2g6-01.SSN.MU__17:27:54#show mlag issu warnings
Please use the MLAG ISSU Compatibility Check tool at https://www.arista.com/en/support/mlag-portal to verify ISSU compatibility.
If you are performing an upgrade, and the Release Notes for the new
version of EOS indicate that MLAG is not backwards-compatible with the
currently installed version (4.25.4M), the upgrade will result in
packet loss.

The configured reload delay of 780 seconds is below the recommended
value of 900 seconds. A longer reload delay allows more time to
rollback an unsuccessful upgrade due to incompatibility.
```

We are using the ansible-avd defaults for it.

**Comment by carlbuchmann** [comment]
@c-po - Yes we are working with EOS PM team to determine the best default recommendation.
At the moment the recommendation would be override the AVD defaults with recommendation from EOS.

**Comment by ClausHolbechArista** [comment]
Fixed in #1143
